docs/tutorials/services/constrain-motion.md

```diff
@@ -0,0 +1,410 @@
+---
+title: "Add Constraints and Transforms to a Motion Plan"
+linkTitle: "Add Motion Constraints"
+weight: 25
+type: "docs"
+description: "Use constraints and transforms with the Motion Service to move robot components in specific ways."
+webmSrc: "/tutorials/videos/motion_constraints.webm"
+mp4Src: "/tutorials/videos/motion_constraints.mp4"
```

Comment:
The mp4 file is too large - also, you need to either retake this footage or crop it to fit a 4:3 aspect ratio. I know not all of them currently exactly match that spec but they are all in landscape mode.

Reply:
the preview is beautiful!